frontend/src/reducers/index.js

```diff
@@ -1,6 +1,8 @@
 import { combineReducers } from 'redux';
 import emptyReducer from './empty';
```

Comment:
Now that we actually have a normal reducer, could you go ahead and delete the empty reducer?